### ISSUE TITLE
connection backend- Lissi-API: editing personal information

### ISSUE THREAD

**Issue by veitmo** [amosproj/amos2022ss04-digital-identity#45]
## User story
1. As a {HR-employee}
2. I want / need {the information in the backend to be forwarded to the Lissi-API, so that the updated information is also forwarded to the blockchain and the digital identity gets updated in the lissi Wallet as well}
3. So that {all information is valid and up-to-date, also to prevent fraud}

## Acceptance criteria
* SDs know how an update of the digital identity on the blockchain is performed
* the functionality of sending all relevant information to Lissi-API and thus updating the information in the blockchain is implemented
* Token in the Lissi App is updated as well

## Definition of done (DoD)
* Added only after week 5
* The same for all features
* Here goes the project specific part

## DoD general criteria
* Feature has been fully implemented
* Feature has been merged into the mainline
* All acceptance criteria were met
* Product owner approved features
* All tests are passing
* Developers agreed to release
